### PR TITLE
Adding Hostname from the server and not as a parameter for stdout

### PR DESCRIPTION
**Pull request by jimonreal**
Cleaning the stdout code a bit, making the hostname not a parameter from stdin but using the one from storagedriver.
Cleaning storagedriver to remove duplicate code lines for the hostname.

### CODE REVIEW

**Comment by rjnagal** [comment]
ok to test

**Comment by rjnagal** [comment]
go vet error:

+go vet github.com/google/cadvisor/...
+godep go test -v -race -test.short github.com/google/cadvisor/...
# github.com/google/cadvisor
./storagedriver.go:91: not enough arguments in call to stdout.New

File: storage/stdout/stdout.go [hunk 1]
```diff
@@ -108,9 +108,9 @@ func (driver *stdoutStorage) Close() error {
 	return nil
 }
 
-func New(namespace string) (*stdoutStorage, error) {
+func New(hostname, namespace string) (*stdoutStorage, error) {
```
Comment (rjnagal) [inline storage/stdout/stdout.go]: namespace not required anymore.

**Comment by mvdan** [comment]
LGTM

**Comment by mvdan** [comment]
Ping @rjnagal

**Comment by rjnagal** [comment]
When cadvisor is running in a container, os.hostname() is not going to return the hostname of the physical host. Is that acceptable for this use-case? It will still be different for each cadvisor instance.

**Comment by jimonreal** [comment]
That will not work. Then I would have to get the HOST hostname from someplace else

**Comment by vishh** [comment]
@jimonreal: You can override the default hostname that docker sets. So this PR should work even with docker.

**Comment by timstclair** [comment]
This PR has been labeled inactive due to lack of recent activity. If it remains in this state for more than 2 weeks, it will be closed. If it is still being developed, please provide a status update.

**Comment by timstclair** [comment]
Closing as **inactive**. Please reopen if development resumes.

**Pull request closed without merging**